website/addons/github/static/githubFangornConfig.js

```diff
@@ -14,7 +14,17 @@ var $osf = require('js/osfHelpers');
 var commandKeys = [224, 17, 91, 93];
 
 function _uploadUrl(item, file) {
-    return waterbutler.buildTreeBeardUpload(item, file, {branch: item.data.branch});
+    // WB v1 update synatx is file path and kind=file
+    // WB v1 upload synatx is parent path and kind=file, name=filename
```

Comment:
minor: s/synatx/syntax/.    formatting suggestion: "WB v1 update syntax is /<file_path>?kind=file"